source/common/common/assert.cc

```diff
@@ -28,15 +28,63 @@ class ActionRegistrationImpl : public ActionRegistration {
   static std::function<void()> debug_assertion_failure_record_action_;
 };
 
+class EnvoyBugRegistrationImpl : public ActionRegistration {
+public:
+  EnvoyBugRegistrationImpl(std::function<void()> action) {
+    ASSERT(envoy_bug_failure_record_action_ == nullptr);
```

Comment:
nit: comment about the reason for this ASSERT, which is related to only supporting one action.

Reply:
done